Makefile

```diff
@@ -1,14 +1,15 @@
 ###########################
 # Configuration Variables #
 ###########################
-ORG := github.com/operator-framework
-PKG := $(ORG)/combo
+PKG := $(shell head -n 1 go.mod | awk '{print $$2}')
 VERSION_PATH := $(PKG)/pkg/version
 GIT_COMMIT := $(shell git rev-parse HEAD)
 DEFAULT_VERSION := v0.0.1
-CONTROLLER_GEN=$(Q)go run sigs.k8s.io/controller-tools/cmd/controller-gen
+CONTROLLER_GEN := $(Q)go run sigs.k8s.io/controller-tools/cmd/controller-gen
 GO_BUILD := $(Q)go build
-PKGS = $(shell go list ./...)
+PKGS := $(shell go list ./...)
+COMBO_VERSION := $(shell git describe || echo $(DEFAULT_VERSION))
```

Comment:
I think `git describe` output would be sufficient without falling back to another variable that increases maintenance over time. We don't have to tackle fixing this in this PR though.

Reply:
Let's get this PR through and talk about what we want in a release. Once we get the first release/tag out I'm comfortable removing this line but without it our combo version breaks